app/src/main/res/values/strings.xml

```diff
@@ -112,9 +112,9 @@
 
     <!-- Edit Profile Screen -->
     <string name="edit_profile_screen_title">Edit your profile</string>
-    <string name="profile_screen_hiking_level_choice_beginner">Beginner</string>
-    <string name="profile_screen_hiking_level_choice_intermediate">Intermediate</string>
-    <string name="profile_screen_hiking_level_choice_expert">Expert</string>
+    <string name="hiking_level_beginner">Beginner</string>
+    <string name="hiking_level_intermediate">Intermediate</string>
+    <string name="hiking_level_expert">Expert</string>
```

Comment:
Question: I like the renaming, but do you think it makes sense to add them to their own section, like "Profile" or something like that, so that its more intuitive that they can also be used outside of the profile screen in the future?

Reply:
You're right. I changed that.